mbzirc_ign/models/mbzirc_fixed_wing/model.sdf.erb

```diff
@@ -0,0 +1,141 @@
+<%
+if !defined?(name) || name== nil || name.empty?
+  $model_name = 'mbzirc_quadrotor'
+else
+  $model_name = name
+end
+%>
+
+<?xml version="1.0"?>
+<sdf version='1.9'>
+  <model name="<%= $model_name%>">
+
+    <!-- Platform base model-->
+    <include merge="true">
+      <uri>model://mbzirc_fixed_wing_base</uri>
+    </include>
+
+    <!-- Multicopter motor for the propeller -->
+    <!--<plugin filename="libignition-gazebo-multicopter-motor-model-system.so"
+      name="ignition::gazebo::systems::MulticopterMotorModel">
+      <robotNamespace>model/<%= $model_name%></robotNamespace>
+      <jointName>propeller_joint</jointName>
+      <linkName>propeller</linkName>
+      <turningDirection>ccw</turningDirection>
+      <timeConstantUp>0.0125</timeConstantUp>
+      <timeConstantDown>0.025</timeConstantDown>
+      <maxRotVelocity>800.0</maxRotVelocity>
+      <motorConstant>8.54858e-06</motorConstant>
+      <momentConstant>0.016</momentConstant>
+      <commandSubTopic>command/motor_speed</commandSubTopic>
+      <motorNumber>0</motorNumber>
+      <rotorDragCoefficient>8.06428e-05</rotorDragCoefficient>
+      <rollingMomentCoefficient>1e-06</rollingMomentCoefficient>
+      <motorSpeedPubTopic>motor_speed/0</motorSpeedPubTopic>
+      <rotorVelocitySlowdownSim>10</rotorVelocitySlowdownSim>
+      <motorType>velocity</motorType>
+    </plugin>-->
+
+    <!-- Joints -->
+    <plugin
+      filename="ignition-gazebo-joint-position-controller-system"
+      name="ignition::gazebo::systems::JointPositionController">
+      <joint_name>flap_left_joint</joint_name>
+      <use_velocity_commands>true</use_velocity_commands>
+      <topic>/model/<%= $model_name%>/joint/left_flap_joint/cmd_pos</topic>
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-joint-position-controller-system"
+      name="ignition::gazebo::systems::JointPositionController">
+      <joint_name>flap_right_joint</joint_name>
+      <use_velocity_commands>true</use_velocity_commands>
+      <topic>/model/<%= $model_name%>/joint/right_flap_joint/cmd_pos</topic>
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-joint-controller-system"
+      name="ignition::gazebo::systems::JointController">
+      <joint_name>propeller_joint</joint_name>
```

Comment:
should we limit the max cmd for the propeller joint so it moves at reasonable fixed wing flight speed? I think the downside is that it's harder to launch the UAV?

Reply:
I'm actually not sure what a "Good" maximum is although I agree we should limit the speed.

Reply:
I found a note here that says 838 max rpm: https://github.com/osrf/uctf/blob/master/models/zephyr_delta_wing_ardupilot_demo/delta_wing.sdf#L137 but don't know the resulting speed is. Let's come back to this when we have more info on the types of fixed wing UAVs that will be used